cfme/fixtures/candu.py

```diff
@@ -38,3 +38,26 @@ def collect_data(appliance, provider, interval='hourly', back='7.days'):
         vm.perf_capture({}, {}.ago.utc, Time.now.utc)\""
         .format(provider.id, repr(vm_name), repr(interval), back))
     return ret.success
+
+
+@pytest.fixture(scope="module")
+def enable_candu_category(appliance):
+    """Enable capture C&U Data for tag category location"""
```

Comment:
Could you please describe how to navigate to that page?

Reply:
@nachandr `location_category.update` will take care of it. `update` method will call to `category` Detail navigation.

Reply:
REQUIRED   
Replace this comment Enable capture C&U Data for tag category location 

with this

Enable capture C&U Data for tag category location by navigating to the Configuration -> Region page.Click 'Tags' tab , select required company category under 'My Company Categories' and enable 'Capture C & U Data' for the category.